core/src/test/scala/zio/logging/LoggerSpec.scala

```diff
@@ -92,6 +94,24 @@ object LoggerSpec extends DefaultRunnableSpec {
             )
           )
       },
+      testM("derive") {
+        val counter = LogAnnotation[Int](
+          "counter",
```

Comment:
Please name these parameters, it will be easier to read.

Reply:
done